scripts/tools/tools-object.js

```diff
@@ -65,8 +65,10 @@ async function convertTools(data) {
           }
         });
       } else {
-        console.log("Repository: " + tool.repository.html_url)
-        console.log("Error: " + validate.errors)
+        console.log('Invalid .asyncapi-tool file.');
+        console.log(`Located in: ${tool.repository.html_url}`);
+        console.log('Validation errors:', JSON.stringify(validate.errors, null, 2));
+        console.log('Not failing, dropping errors for further investigation');
```

Comment:
Maybe `console.error` will be better?

Reply:
Yeah, `console.error` is better but I think what we discussed in the previous meetings that we should notify about each error in the Slack notification channel. That functionality is left here. Will soon add it to the upcoming PR.

Reply:
yeah, getting stuff to slack will be tricky, as normal flow will not work. We will need to use GitHub Action `core` library to set output for GH Action workflow, so we can react and drop message to slack. I can add it in this PR if ya want

Reply:
But if you want to use Slack notifications, I would suggest you to use REST API of Slack (if exists 😅) as the log messages are present in the script files and making a API call from that script file to create notifications is lot more easier and reliable instead of using Github Actions.

Reply:
But if you want to use Slack notifications, I would suggest you to use REST API of Slack (if exists 😅) as the log messages are present in the script files and making a API call from that script file to create notifications is lot more easier and reliable instead of using Github Actions.

Reply:
For reference, you can read this documentation - https://api.slack.com/messaging/sending

Reply:
If you want you can add integration with slack. If not, you can do it in another PR :)

Reply:
now that I think about it, better would be to add integration later after tools view is released because we will get slack notifications now that will annoy us (there is already one file that is invalid in one repo). I'll add issue so we don't forget